NugetLightRequest.cs

```diff
@@ -1486,44 +1595,61 @@ into newGroup
                     // if exact version is not required and allow prerelease is false, we will have to filter out prerelease version
                     pkgs = from p in pkgs where string.IsNullOrWhiteSpace(p.Version.SpecialVersion) select p;
                 }
-             
-                pkgs = FilterOnContains(pkgs);
-                pkgs = FilterOnTags(pkgs);
 
-                var results = FilterOnVersion(pkgs, requiredVersion, minimumVersion, maximumVersion, minInclusive, maxInclusive)
-                    .Select(pkg => new PackageItem {
-                        Package = pkg,
-                        PackageSource = source,
-                        FastPath = MakeFastPath(source, pkg.Id, pkg.Version.ToString())
-                    });
+                pkgs = PackageFilterUtility.FilterOnContains(pkgs, Contains.Value);
```

Comment:
>pkgs = PackageFilterUtility.FilterOnContains(pkgs, Contains.Value); [](start = 16, length = 67)

Is contains always specified? if not, consider adding a check on Contains similar to below FilterOnTag check.

Reply:
Check is in PackageFilterUtility.FilterOnContains